parsec/core/fs/remote_loader.py

```diff
@@ -850,6 +855,8 @@ async def _vlob_create(
                 sequester_authority_certificate=rep.sequester_authority_certificate,
                 sequester_services_certificates=rep.sequester_services_certificates,
             )
+        elif isinstance(rep, VlobCreateRepSequesterRejected):
+            raise VlobSequesterRejectedError(id=entry_id)
```

Comment:
no manifest provided ?

Don't we want to catch this in the caller `upload_manifest` so that we have the manifest object at hand ?